packages/app/src/cli/services/deploy/upload.test.ts

```diff
@@ -314,441 +274,26 @@ describe('uploadFunctionExtensions', () => {
       const uploadUrl = 'test://test.com/moduleId.wasm'
       extension.outputPath = joinPath(tmpDir, 'index.wasm')
       await writeFile(extension.outputPath, '')
-      const uploadURLResponse: UploadUrlGenerateMutationSchema = {
-        data: {
-          uploadUrlGenerate: {
-            headers: {},
-            maxSize: '200',
-            url: uploadUrl,
-            moduleId: 'module-id',
-          },
+      const uploadURLResponse: FunctionUploadUrlGenerateResponse = {
```

Comment:
I think this test is the same as `errors if the upload of the wasm errors`. I don't see a compilation request in the code anymore.